cgmanifest.json

```diff
@@ -1442,6 +1442,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "cargo2rpm",
```

Comment:
why are cargo2rpm and rust-packaging being added?

Reply:
initially they were added because the other distro's version of mdevctl spec used them to package the rpm, but i found that they didn't work well and got rid of them.  neither should be included in the PR anymore.

Reply:
oops ... will remove it from cgmanifest

Reply:
close and reopen the PR seemed to fix whatever was going on.  cargo2rpm and rust-packaging should be gone now.